SUMMARY.md

```diff
@@ -0,0 +1,34 @@
+## Table of Contents
+
+* [Connecting to Database](content/ContectingToDatabase/ConnectingToDatabase.q.md) ** - minutes**
+    > Learn how you can use a specific class or command to establish a connection between PHP and PostgreSQL.
+
+* [PHP Data Objects](content/PDOClasses/PDOClasses.q.md) ** - minutes**
+    > In this lesson, you will learn about PHP Data Objects (`PDO`) classes and how to execute queries using `PDO::query()`.
+
+* [Prepared Statement](content/PreparedStatement/PreparedStatement.q.md) ** - minutes**
```

Comment:
The SQL injection is mentioned in the last part of the course, I think we should move this there. After maybe.
Discuss SQL, then discuss how to prevent it.

Reply:
This noted. SQL injection will be introduced in the lesson `Prepared Statement`. Thanks.